src/components/Footer/Footer.tsx

```diff
@@ -56,6 +62,24 @@ function Footer({ socialLinks, additionalLinks }: Props) {
               ))}
           </ul>
         </Col>
+        <Col size={6}>
+          <p className="u-align--left">
+            Ubuntu and Canonical are registered trademarks of Canonical Ltd.
+            <br></br>
+            Powered by{" "}
+            <a href="https://www.ubuntu.com/kubernetes">Charmed Kubernetes</a>
+          </p>
+          <hr></hr>
```

Comment:
This `hr` should only show if there are optional links. Also please use `<hr />`